### ISSUE TITLE
Need better code hinting

### ISSUE THREAD

**Issue by chrisvelevitch** [HaxeFoundation/intellij-haxe#899]
When I `cmd+mouseover` (on OSX) a variable name, all I get is:-
```
field "fieldName" [filenameWith.ext]
```
I'm used to IntelliJ plugins that also include the type of the field as well. Something like:-
```
private var my.package.path.ThisClassName.fieldName:TypeName (filenameWith.ext)
```
where `filenameWith.ext` is the actual file name where this field was orginailly defined (important when the variable is inherited from another class in the hierarchy). If it was defined in the current file, then it can be left out.

**Comment by EricBishton** [comment]
Which version(s) of IDEA and the plugin are you using?  And, do you have compiler completions turned on?

**Comment by chrisvelevitch** [comment]
I'm using IntelliJ 2018.3, Haxe Toolkit v1.1 and Haxe Checkstyle v1.1 and in `Preferences->Editor->General->Code Completion`, completion is on.

**Comment by m0rkeulv** [comment]
Solved in 1.4.11
